aiida/cmdline/commands/cmd_computer.py

```diff
@@ -155,26 +156,26 @@ def _computer_create_temp_file(transport, scheduler, authinfo):  # pylint: disab
 
     with tempfile.NamedTemporaryFile(mode='w+') as tempf:
         fname = os.path.split(tempf.name)[1]
-        echo.echo('  `-> Creating the file {}...'.format(fname))
+        echo.echo('  -> Creating the file {}...'.format(fname))
         remote_file_path = os.path.join(workdir, fname)
         tempf.write(file_content)
         tempf.flush()
         transport.putfile(tempf.name, remote_file_path)
-    echo.echo('  `-> Checking if the file has been created...')
+    echo.echo('  -> Checking if the file has been created...')
     if not transport.path_exists(remote_file_path):
         echo.echo_error('* ERROR! The file was not found!')
         return False
 
-    echo.echo('      [OK]')
-    echo.echo('  `-> Retrieving the file and checking its content...')
+    echo.echo('     [OK]')
+    echo.echo('  -> Retrieving the file and checking its content...')
 
     handle, destfile = tempfile.mkstemp()
     os.close(handle)
     try:
         transport.getfile(remote_file_path, destfile)
         with io.open(destfile, encoding='utf8') as dfile:
             read_string = dfile.read()
-        echo.echo('      [Retrieved]')
+        echo.echo('     [Retrieved]')
```

Comment:
maybe remove this one and just put `[OK]` instead of `[Content OK]` on same line.